src/main/java/tira/task/TaskList.java

```diff
@@ -0,0 +1,133 @@
+package tira.task;
+
+import tira.Ui;
+import tira.TiraException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.io.PrintWriter;
+import java.io.IOException;
+
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+    private final PrintWriter printer = new PrintWriter(System.out);
+    private static DateTimeFormatter IN_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    private final Ui ui = new Ui();
+
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public ArrayList<Task> getTasks() {
+        return this.tasks;
+    }
+
+    public void markTask(String command, String[] splitCommand) throws TiraException {
+        if (splitCommand.length < 2) {
+            throw new TiraException("MRAW?? WHERE IS THE TASK?");
+        }
+        int currNum = Integer.parseInt(splitCommand[1]) - 1;
+        tasks.get(currNum).markStatus();
+        Task currTask = tasks.get(currNum);
```

Comment:
I'm not too sure I like the naming of `currNum` and `currTask`. Perhaps a more intuitive name could be used here, such as `index` and `currentTask`.